packages/vite/src/utils.ts

```diff
@@ -29,3 +28,18 @@ function hash(keyParts: Array<string | NodeJS.ArrayBufferView>) {
   }
   return hash.digest('hex')
 }
+
+export const generateCacheID = (path: string) => createHash('sha1').update(path).digest('hex')
+
+export const checksumFile = (algorithm: string, path: string) => {
```

Comment:
Do we need a new `checksumFile` method or can we reuse `generateImageID`, which does something similar already? Also, this is more of a hash than a checksum, so perhaps we could rename it

Reply:
I renamed the function and the property in the cache metadata file.